sqlmesh/core/engine_adapter/bigquery.py

```diff
@@ -469,8 +470,55 @@ def _create_table_properties(
         if clustered_by:
             properties.append(exp.Cluster(expressions=[exp.column(col) for col in clustered_by]))
 
+        if table_description:
+            properties.append(
+                exp.Property(
+                    this=exp.Identifier(this="description", quoted=False),
+                    value=exp.Literal.string(table_description),
+                )
+            )
+
         for key, value in (table_properties or {}).items():
-            properties.append(exp.Property(this=key, value=value))
+            properties.append(exp.Property(this=key, value=value.copy()))
+
```

Comment:
We may be able to reuse https://github.com/TobikoData/sqlmesh/blob/c0cf31f821ae4768ed18f7d5b79d204ebbc8492b/sqlmesh/core/engine_adapter/mixins.py#L216 here

Reply:
@eakmanrq If we pull this out of the mixin so it can be reused, where should it go?

Reply:
You could move it to base.